examples/custom_solvers.jl

```diff
@@ -79,25 +79,20 @@ scfres = self_consistent_field(basis;
 
 # ## Customizing the convergence criterion
 # Here is an example of a defining a custom convergence criterion and specifying
-# it using the `is_converged` keyword to `self_consistent_field`.
+# it using the `is_converged` callback keyword to `self_consistent_field`.
 
-mutable struct ScfConvergenceStress
-    tolerance
-    previous_stress
-end
-ScfConvergenceStress(tolerance) = ScfConvergenceStress(tolerance, nothing)
-function (conv::ScfConvergenceStress)(info)
-    # If first iteration clear a potentially cached previous stress
-    info.n_iter ≤ 1 && (conv.previous_stress = nothing)
-    (; basis, ψ, occupation, eigenvalues, εF, ρout) = info
-    stress = compute_stresses_cart((; basis, ψ, occupation, eigenvalues, εF, ρ=ρout))
-    error = isnothing(conv.previous_stress) ? NaN : norm(conv.previous_stress - stress)
-    conv.previous_stress = stress
-    error < conv.tolerance
+function my_convergence_criterion(info)
+    tol = 1e-10
+    if last(info.history_Δρ) > 10sqrt(conv.tolerance)
+        return false  # The ρ change should also be small to avoid the SCF being just stuck
+    end
```

Comment:
Not even sure I would do this in this simple example.

Reply:
Removed in [0a89022](https://github.com/JuliaMolSim/DFTK.jl/pull/811/commits/0a8902294752f0f345849f08b3b448a395c9eecd)